<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>380</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>460</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="projectButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Проектирование</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prototypeButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Протот-е</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sienceButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Исследования</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>270</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Пуск. подг.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buildButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Сборка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Тестирование</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_project_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_test_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_science_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_prototype_1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_assembly_1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_start_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>250</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>371</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="slider6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>580</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider7">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>620</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>540</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>660</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Рестарт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>530</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Запуск цикла</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>380</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>420</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>460</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>540</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>580</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>660</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bossButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>650</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Руководитель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="managerButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>690</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Менеджер</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brigButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>730</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Бригадир</string>
    </property>
   </widget>
   <widget class="QPushButton" name="controlButton">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>ОТК</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_control_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_project_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_prototype_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_assembly_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_control_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_science_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_test_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb_start_1">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>740</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>700</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="slider9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>700</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>740</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_yellow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_tripple.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_yellow_left.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_yellow_up.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_yellow.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/stase/Pictures/ДЗ геймдев/Новая папка/Array_yellow_down.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
